README.md

```diff
@@ -223,7 +228,9 @@ one with `8786` being open. Other than that, no additional configurations should
 python runner.py --wf ttcom --executor dask/lxplus
 ```
 
-jobs automatically splitted to 100 files per jobs to avoid job failure due to crowded cluster on lxplus with the naming scheme `hist_$workflow_$json_$index.coffea`, combined in plotting scripts
+jobs automatically splitted to 50 files per jobs to avoid job failure due to crowded cluster on lxplus with the naming scheme `hist_$workflow_$json_$dictindex_$fileindex.coffea`, combined in plotting scripts.
+
+To deal unstable condor cluster and dask worker on lxplus, you can resubmit failure jobs via `--index $dictindex,$fileindex` option. `$dictindex` refers the index in the `.json dict`, `$fileindex` refers to the index of the file list splitted to 50 files per dask-worker. The total numbers of files of each dict could be computed by `math.ceil(len($filelist)/50)` The job will start from the corresponding index. 
```

Comment:
suggest to modify:
```
To deal with unstable condor cluster and dask worker on lxplus, you can resubmit failure jobs via `--index $dictindex,$fileindex` option. `$dictindex` refers to the index in the `.json dict`, `$fileindex` refers to the index of the file list split to 50 files per dask-worker. The total number of files of each dict can be computed by `math.ceil(len($filelist)/50)` The job will start from the corresponding indices.
```
Could you also please add the optimal scaleout and chunk size for lxplus? i.e. `-s 50 --chunk 50000`, preceded by an exclamation mark symbol?

Reply:
Maybe switching from `cvmfs-contrib/github-action-cvmfs@v2` to `cvmfs-contrib/github-action-cvmfs@v3` helps? has been released couple of days ago (see [here](https://github.com/cvmfs-contrib/github-action-cvmfs)), I have to check if this is connected to the python versions or has nothing to do with it

Reply:
> suggest to modify:
> 
> ```
> To deal with unstable condor cluster and dask worker on lxplus, you can resubmit failure jobs via `--index $dictindex,$fileindex` option. `$dictindex` refers to the index in the `.json dict`, `$fileindex` refers to the index of the file list split to 50 files per dask-worker. The total number of files of each dict can be computed by `math.ceil(len($filelist)/50)` The job will start from the corresponding indices.
> ```
> 
> Could you also please add the optimal scaleout and chunk size for lxplus? i.e. `-s 50 --chunk 50000`, preceded by an exclamation mark symbol?

Ok


> Maybe switching from `cvmfs-contrib/github-action-cvmfs@v2` to `cvmfs-contrib/github-action-cvmfs@v3` helps? has been released couple of days ago (see [here](https://github.com/cvmfs-contrib/github-action-cvmfs)), I have to check if this is connected to the python versions or has nothing to do with it

I don't think the git actions and python versions related to scaleout on lxplus. Did I misunderstand something?

Reply:
Yeah I placed that comment incorrectly, it was meant for the issue with CI pipeline + python 3.10